crates/next-core/src/manifest.rs

```diff
@@ -75,25 +150,30 @@ impl ContentSource for DevManifestContentSource {
         path: &str,
         _data: turbo_tasks::Value<ContentSourceData>,
     ) -> Result<ContentSourceResultVc> {
-        let manifest_content = match path {
+        let manifest_file = match path {
             "_next/static/development/_devPagesManifest.json" => {
                 let pages = &*self_vc.find_routes().await?;
 
-                serde_json::to_string(&serde_json::json!({
+                File::from(serde_json::to_string(&serde_json::json!({
                     "pages": pages,
-                }))?
+                }))?)
+                .with_content_type(APPLICATION_JSON)
+            }
+            "_next/static/development/_buildManifest.js" => {
+                let build_manifest = &*self_vc.create_build_manifest().await?;
+
+                File::from(build_manifest.as_str()).with_content_type(APPLICATION_JAVASCRIPT)
```

Comment:
Nit:
```suggestion
                File::from(build_manifest.as_str()).with_content_type(APPLICATION_JSON_UTF_8)
```

Reply:
Was that supposed to be `APPLICATION_JAVASCRIPT_UTF_8`?

Reply:
Yes 🤦

Reply:
done